### PR TITLE
takari-team-maven 0.9.0 (new formula)

### PR DESCRIPTION
**Pull request by chrisalbright**
http://takari.io/book/20-introduction.html#installing-team

### CODE REVIEW

File: Library/Formula/takari-team-maven.rb [hunk 1]
```diff
@@ -0,0 +1,53 @@
+class TakariTeamMaven < Formula
+  desc "Takari Maven Distribution"
+  homepage "http://takari.io/"
+  url "http://repo1.maven.org/maven2/io/takari/takari-team-maven/0.9.0/takari-team-maven-0.9.0.tar.gz"
+  # version "0.9.0"
```
Comment (DomT4) [inline Library/Formula/takari-team-maven.rb]: Should be safe to remove this line.

File: Library/Formula/takari-team-maven.rb [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    # file will be found relative to it
+    Pathname.glob("#{libexec}/bin/*") do |file|
+      next if file.directory?
+      basename = file.basename
+      next if basename.to_s == "m2.conf"
+      (bin/basename).write_env_script file, Language::Java.overridable_java_home_env
+    end
+  end
+
+  conflicts_with "mvnvm", :because => "also installs a 'mvn' executable"
```
Comment (DomT4) [inline Library/Formula/takari-team-maven.rb]: This should be one blank line under the last `depends_on`

File: Library/Formula/takari-team-maven.rb [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    EOS
+    (testpath/"src/main/java/org/homebrew/MavenTest.java").write <<-EOS.undent
+      package org.homebrew;
+      public class MavenTest {
+        public static void main(String[] args) {
+          System.out.println("Testing Maven with Homebrew!");
+        }
+      }
+    EOS
+    system "#{bin}/mvn", "compile"
```
Comment (DomT4) [inline Library/Formula/takari-team-maven.rb]: You'll need to pass `ENV.java_cache` at the top of `test do` here.

File: Library/Formula/takari-team-maven.rb [hunk 4]
```diff
@@ -0,0 +1,53 @@
+class TakariTeamMaven < Formula
+  desc "Takari Maven Distribution"
+  homepage "http://takari.io/"
+  url "http://repo1.maven.org/maven2/io/takari/takari-team-maven/0.9.0/takari-team-maven-0.9.0.tar.gz"
```
Comment (DomT4) [inline Library/Formula/takari-team-maven.rb]: Looks like this can use `https://`

**Comment by bfontaine** [comment]
Ping re Dom’s comments?

**Comment by DomT4** [comment]
Closing as dead. Please feel free to resubmit with comments addressed if desired. Thank for your submission to Homebrew!

**Pull request closed without merging**